src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -1,47 +1,51 @@
 /** @jsx React.DOM */
 
+"use strict";
 
 var React = require("react/addons");
 
-module.exports = React.createClass({
```

Comment:
I kinda prefer this style to the new one. It forces you to only have one component/file and removes any possible repetition.